app/config.json.sample

```diff
@@ -13,5 +13,6 @@
     ],
     "addingRelationDialog": [
         "Pet"
-    ]
+    ],
+    "pageLimit": 10
```

Comment:
Anything works here. I'll change that asap.
On Jun 15, 2016 8:42 PM, "Nachi Ueno" <notifications@github.com> wrote:

> In app/config.json.sample
> <https://github.com/cloudwan/gohan_webui/pull/56#discussion_r67221264>:
>
> > @@ -13,5 +13,6 @@
> >      ],
> >      "addingRelationDialog": [
> >          "Pet"
> > -    ]
> > +    ],
> > +    "pageLimit": 10
>
> I prefer more bigger default value. How about 100?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cloudwan/gohan_webui/pull/56/files/76f5e542696f242aadef20780e2e8a730a1e44b2#r67221264>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APliZmv8JTkO9XBcOTaFIPe0VJXUOjoyks5qMEeegaJpZM4I2EpS>
> .
>